add manual for Intech TEK2 (#686)
mixxxdj/manual: source/hardware/controllers/intech_TEK2.rst

Intech Studio - Grid (TEK2)
===========================

-  `Intech Studio <https://intech.studio/>`__

> .. figure:: ../../_static/controllers/intech_TEK2.png

Reviewer: file names are case-sensitive, so either this
```suggestion
.. figure:: ../../_static/controllers/intech_tek2.png
```
or you need to rename the png.

   :align: center
   :width: 80%
   :figwidth: 80%
   :alt: Intech Studio TEK2
   :figclass: pretty-figures

   TEK2

.. versionadded:: 2.4.2
TEK2 is a modular controller with 5-point multicolor LED feedback and massive knobs with a two-state switch (pressed & released) in addition to their smooth endless turn. The programmable ‘5-bit display’ indicator can give feedback on centered or starting value positions and highlight parameter states.

Intech Grid modules are modular and fully configurable, allowing you to create your own DJ MIDI controller.

* `TEK2 product page <https://intech.studio/shop/tek2>`__
* `Discord support channel <https://discord.com/invite/ntrzhxS>`__

TEK2 and Scratch
----------------

This basic mapping provides jogwheel and scratch functionality in Mixxx.

> The first row of buttons is mapped to Play and Cue on each channel.

Reviewer: ```suggestion
The row of buttons below the jogwheels is mapped to Play and Cue on each channel.
The buttons in the bottom row are unmapped and can be customized with the Mixxx MIDI mapping wizard.
```

Setting up the controller
-------------------------

To make this mapping work, follow these steps:

1. Download and open the `Grid Editor <https://intech.studio/ae/products/grid-editor/>`__.
2. In the Grid Editor, access the Profile Cloud where you can load and save configurations to our Modules.
3. Search for 'TEK2 DJ setup', then load and store it to your TEK2.

> 4. Open up MIxxx, go to :menuselection:`Preferences` --> :menuselection:`Controllers` --> :menuselection:`Intech TEK2` and select the 'Intech TEK2' mapping.

Reviewer: typo from my suggestion..
```suggestion
4. Open up Mixxx, go to :menuselection:`Preferences` --> :menuselection:`Controllers` --> :menuselection:`Intech TEK2` and select the 'Intech TEK2' mapping.
```

Intech Studio - Grid (TEK2)
===========================

-  `Intech Studio <https://intech.studio/>`__

> .. versionadded:: 2.4

Reviewer: ```suggestion
.. versionadded:: 2.4.2
```

> 

Reviewer: How about adding the info from https://intech.studio/shop/tek2?

```suggestion
TEK2 is a modular controller with 5-point multicolor LED feedback and massive knobs with a two-state switch (pressed & released) in addition to their smooth endless turn. The programmable ‘5-bit display’ indicator can give feedback on centered or starting value positions and highlight parameter states.

Intech Grid modules are modular and fully configurable, allowing you to create your own DJ MIDI controller.

* `TEK2 product page <https://intech.studio/shop/tek2>`
* `Discord support channel <https://discord.com/invite/ntrzhxS>`

```

TEK2 and Scratch
----------------

> This simple configuration is created for the Intech Studio `TEK2 module <https://intech.studio/shop/tek2>`__, 
> enabling the use of the Jogwheel and Scratch functionality in MIXXX software. 
> Our Grid modules are modular and fully configurable, allowing you to create your own DJ MIDI controller.
> We typically use the `TEK2 <https://intech.studio/shop/tek2>`__, `PBF4 <https://intech.studio/shop/pbf4>`__, and `PO16 <https://intech.studio/shop/po16>`__ modules, which together form a straightforward combination to control any DJ software. 
> However, everyone's needs and ideas for plug together Grids are different, 
> so creating custom mappings in MIXXX is entirely up to you—it's really easy!

Reviewer: ```suggestion
This basic mapping provides jogwheel and scratch functionality in Mixxx.
```

Setting up the controller
-------------------------

> To make this mapping work, follow these settings:
> 
> 1. Download and open the `Grid Editor <https://intech.studio/ae/products/grid-editor/>`__.
> 2. In the Grid Editor, access the Profile Cloud where you can load and save configurations to our Modules.
> 3. Search for 'TEK2 DJ setup', then load and store it to your TEK2.
> 4. Open up MIXXX, go to Preferences/Controllers, and select the 'Intech TEK2' profile.
> 
> That's it! You are now able to use the jog wheel and create your custom mapping.
> 
> 
> If you have any questions or need help with the setup, 
> join our `Discord channel <https://discord.com/invite/ntrzhxS>`__
> and we will assist you!

Reviewer: ```suggestion
To make this mapping work, follow these steps:

1. Download and open the `Grid Editor <https://intech.studio/ae/products/grid-editor/>`__.
2. In the Grid Editor, access the Profile Cloud where you can load and save configurations to our Modules.
3. Search for 'TEK2 DJ setup', then load and store it to your TEK2.
4. Open up MIxxx, go to :menuselection:`Preferences` --> :menuselection:`Controllers` --> :menuselection:`Intech TEK2` and select the 'Intech TEK2' mapping.

That's it! You are now able to use the jog wheel and create your custom mapping.

```

> .. figure:: ../../_static/controllers/intech_dj.png

Reviewer: Please use an image of a single TEK2.

Author: Picture changed in next commits
